plugins/labhub.py

```diff
@@ -136,3 +136,28 @@ def create_issut_cmd(self, msg, match):
             return ('Can\'t create an issue for a repository that does not '
                     'exist. Please ensure that the repository is available '
                     'and owned by coala.')
+
+    @re_botcmd(pattern=r'^unassign\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)/issues/(\d+)',  # Ignore LineLengthBear, PyCodeStyleBear
+               flags=re.IGNORECASE)
+    def unassign_cmd(self, msg, match):
+        """Unassign from an issue."""  # Ignore QuotesBear
+        org = match.group(2)
+        repo_name = match.group(3)
+        issue_number = match.group(4)
+
+        user = msg.frm.nick
+
+        try:
+            assert org == self.GH_ORG_NAME or org == self.GL_ORG_NAME
+        except AssertionError:
+            return 'Repository not owned by our org.'
+
+        try:
+            iss = self.REPOS[repo_name].get_issue(int(issue_number))
```

Comment:
Can we apply the else fix here as well https://github.com/coala/corobo/pull/71/commits/e9a6b23cbc3993630a0936543643870c85e661f3#diff-dd3a6bcfd3fdd49afb75cd26d1429417R47